Classes/Repository/RepositoryViewController.swift

```diff
@@ -48,6 +48,30 @@ PrimaryViewController {
         feed.viewDidLoad()
         feed.adapter.dataSource = self
         title = "\(repo.owner)/\(repo.name)"
+        
+        if repo.hasIssuesEnabled {
+            navigationItem.rightBarButtonItem = UIBarButtonItem(
```

Comment:
@BasThomas You mean like this code here 😉

I know I need to move it to the action sheet, but the logic is all here to have a "New Issue" button for the repo view controller - Just don't merge master into this branch 😂